HomeSol/Home_5/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Home_5;
+using System;
+using System.Text.RegularExpressions;
+
+namespace MyApp // Note: actual namespace depends on the project name.
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            
+
+            Student[] group1 = new Student[5]; // Generating group 1
+            for (int i = 0; i <= group1.Length - 1; i++)
+            {
```

Comment:
It is better to use a separate class for that. Please, watch a class video recording OOP where we discussed it.